Cargo.toml

```diff
@@ -32,4 +32,5 @@ winapi = { version = "0.3.9", features = [
     "processenv",
     "processthreadsapi",
     "memoryapi",
+    "winnt"
```

Comment:
What functionality reququires this feaure? Maybe I missed something during the review.

Reply:
```
use winapi::um::winnt::{
    MEMORY_BASIC_INFORMATION, PROCESS_QUERY_INFORMATION, PROCESS_VM_OPERATION, PROCESS_VM_READ,
    PROCESS_VM_WRITE,
};
```